examples/model_assisted_labeling/video_mal.ipynb

```diff
@@ -0,0 +1,361 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    !pip install labelbox
</pre><p><code>pip install -q</code> is better  because it will print errors but also not log every lib that is downloaded. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/375/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>